src/slang_frontend.cc

```diff
@@ -32,6 +32,12 @@
 #include "async_pattern.h"
 #include "variables.h"
 
+#ifdef log_debug
```

Comment:
Thanks, would
```
#ifndef log_debug
using Yosys::log_debug
#endif
```
work too?

If yes, I'd prefer that. We can put it into `slang_frontend.h` with other `using` declarations.

Reply:
Yes, that is passing. Have updated PR